source/_integrations/vicare.markdown

```diff
@@ -66,10 +66,9 @@ name:
   default: ViCare
   type: string
 heating_type:
-  description: One of `generic`, `gas`, `heatpump` or `fuelcell`. Specifying the heating_type provides additional attributes and sensors specific for the heating system.
-  required: false
+  description: One of `auto`, `gas`, `heatpump` or `fuelcell`. Specifying the heating_type provides additional attributes and sensors specific for the heating system.
+  required: true
```

Comment:
According to the configuration schema, this is actually optional.
Can you point me to the source of your change?

Reply:
I received on my HASS that the conf was not complete and that this was a required field. this happend after the upgrade to the latest version. I saw also instant that the general was gone and that the auto was introduced.

Reply:
this is the error i received when i checked the conf:
Invalid config for [vicare]: value must be one of ['auto', 'fuelcell', 'gas', 'heatpump', 'oil', 'pellets'] for dictionary value @ data['vicare']['heating_type']. Got None. (See /config/configuration.yaml, line 130).

Reply:
That is because you've passed an invalid value. Where does it state it is an required value?

Reply:
My Config was without heat type and after the upgrade of hass it said my Config was invalid

Reply:
I cannot find any evidence of the code change that describes that behavior, could you point me to it?